sections/leaderboard/TraderHistory.tsx

```diff
@@ -189,9 +189,12 @@ const TraderHistory: FC<TraderHistoryProps> = memo(
 										accessor: 'pnl',
 										Cell: (cellProps: CellProps<typeof data[number]>) => (
 											<PnlContainer>
-												<ColorCodedPrice
+												<Currency.Price
+													currencyKey="sUSD"
 													price={cellProps.row.original.pnl}
-													$value={cellProps.row.original.pnl}
+													sign="$"
+													conversionRate={1}
```

Comment:
I think the `currencyKey`, `sign` and `conversionRate` props can be omitted, since they are set to their default values. Also, you can omit the `={true}` after the `colored` prop.